lambda/fulfillment/lib/middleware/lex.js

```diff
@@ -191,6 +214,57 @@ function buildImageResponseCardV2(response) {
     return imageResponseCardV2;
 }
 
+function buildInteractiveMessageElements(elements){
+    return elements.map(x => ({title: x.text}));
+}
+
+function buildInteractiveMessageTemplate(response){
+    response = limitInteractiveMessagesDisplayTextLength(response);
+
+    if(response.message.length > 100){
+        qnabot.log("WARNING: Truncating message content to Interactive Message Title limit of 100 characters");
+    }
+
```

Comment:
Future consideration: for the title and subtitle cases being truncated due to character limits (100 and 200 respectively), one option is to make this more apparent (other than the log), so that it can be addressed if needed and also user noticing truncation rather than wondering about spelling/grammar error. For example, if we truncated, we could place indicators such as `...` or `[truncated]` at the end of the text (while such a technique would use several extra characters).